src/Annotation/ApiResource.php

```diff
@@ -52,7 +52,8 @@
  *     @Attribute("shortName", type="string"),
  *     @Attribute("subresourceOperations", type="array"),
  *     @Attribute("swaggerContext", type="array"),
- *     @Attribute("validationGroups", type="mixed")
+ *     @Attribute("validationGroups", type="mixed"),
+ *     @Attribute("sunset", type="string")
```

Comment:
Can you move it after `subresourceOperations`? (alpha order)

Reply:
should now be the case